src/context/SettingsContext.jsx

```diff
@@ -4,6 +4,7 @@ import { BTC } from '../utils'
 const localStorageKey = 'jm-settings'
 
 const initialSettings = {
+  theme: 'light',
```

Comment:
This one's a bit tricky, because the default depends on the system preferences of the user. (see also index,html)

```suggestion
  theme: window.matchMedia('(prefers-color-scheme: dark)').matches ? window.JM.THEMES[1] : window.JM.THEMES[0];
```

Reply:
I think we don't even need this as it already gets set [in `index.html`](https://github.com/joinmarket-webui/joinmarket-webui/blob/master/public/index.html).

Reply:
Yes, I did set it do 'light' even if it is done in index.html -> kind of has a documentary purpose. But I can remove it, if you want to. I kind of like to see all the default settings (unless overwritten, of course).

Reply:
I think it makes sense to have it removed to not duplicate code that might get out of sync.

Reply:
Removed! Thank you both for your inputs!